velox/exec/ContainerRowSerde.cpp

```diff
@@ -814,8 +814,8 @@ void ContainerRowSerde::serialize(
 void ContainerRowSerde::deserialize(
     ByteStream& in,
     vector_size_t index,
-    BaseVector* result) const {
```

Comment:
Why is this change needed to the deserialize API ?

Reply:
Reverted, thanks for the suggestion! It was initially added because of the call to deserialize from extractComplexType.